lock-api/src/main/java/com/palantir/lock/client/BatchingIdentifiedAtlasDbTransactionStarter.java

```diff
@@ -42,7 +42,7 @@ private BatchingIdentifiedAtlasDbTransactionStarter(
         this.autobatcher = autobatcher;
     }
 
-    static BatchingIdentifiedAtlasDbTransactionStarter create(
+    public static BatchingIdentifiedAtlasDbTransactionStarter create(
             LockLeaseService lockLeaseService, StartTransactionsLockWatchEventCache lockWatchEventCache) {
```

Comment:
in hindsight: if STLWEC is only used here, does it make sense to have the factory take a full LWEC?